### DISCUSSION TITLE
Video file as an attachment

### DISCUSSION THREAD

**Discussion by mmdislam** [PHPMailer/PHPMailer#3053] [General]
Can a video file be added as attachment? I have tried and did not succeed. Any idea?

**Comment by Synchro** [comment]
You can use any kind of file as an attachment; PHPMailer doesn't care. You don't say what you've done and how it failed to work. How it will be handled by clients will vary a lot – some might allow it to play inline, others may only offer a download. Generally though, attaching very large files is more of a problem; it's much better to put it on a web page and email a link to it.

**Reply by mmdislam** [reply]
Successfully sent the video as an attachment. Initially encountered an error stating "SERVER -> CLIENT: 552 5.3.4 Message exceeds max size of 10485760 bytes" due to the video file size being 9 MB. Reduced the file size to 4 MB, and now it's working without any issues.
